src/main/resources/scripts/cleanup/non_prod_primary_and_replica_data_cleanup.sql

```diff
@@ -0,0 +1,101 @@
+--
+-- RWA-2482 cleanup old tasks from primary task database
+--
+--Example for calling the function
+--select cft_task_db.task_cleanup_between_dates_primary('2023-04-04 11:00'::TIMESTAMP,'2023-04-04 11:01'::TIMESTAMP);
+--
+
+CREATE OR REPLACE FUNCTION cft_task_db.task_cleanup_between_dates_primary(create_from TIMESTAMP,create_to TIMESTAMP)
+RETURNS VOID
+LANGUAGE plpgsql
+AS $function$
+DECLARE
+    task_id_var TEXT;
+BEGIN
+    -- Loop through the rows of the tasks table
+    FOR task_id_var IN SELECT task_id FROM cft_task_db.tasks WHERE created >= create_from AND created <= create_to order by created limit 2000 LOOP
```

Comment:
Is this limit what you think is correct following the deletions you did?

Reply:
Yes Ravi... If the limit is more than 2000 the triggers are chocking the replica DB. CPU usage is going beyond 95%.

Reply:
Ok sounds good then.